website/blog/2024-10-28-kustomize-vs-helm/index.mdx

```diff
@@ -0,0 +1,179 @@
+| **Customization flexibility**  | High for environment-specific changes             | High for full application lifecycle                                  |
+| **Pre-built ecosystem**        | None                                              | Extensive (large library of pre-built charts)                        |
+| **CI/CD integration**          | Ideal for GitOps                                  | Well-suited for traditional CI/CD workflows and GitOps               |
+| **Complexity of logic**        | Low (no logic or scripting)                       | Medium to High (templating logic, hooks, and scripts)                |
+
+### Kustomize vs Helm: Method of operation
+
+Helm uses a templating engine, allowing users to pass values into predefined templates for dynamic, repeatable configurations. However, this introduces complexity, as the unrendered templates can be difficult to read and debug. 
+
+Kustomize, in contrast, focuses on declarative configurations with plain Kubernetes manifests, making it more intuitive and readable. While Kustomize simplifies configuration management, it lacks some of Helm’s imperative features, like state management, which can be useful in more complex scenarios.
+
+### Kustomize vs Helm:  Ease of use
+
+Helm is often considered easier for packaging entire applications, thanks to its extensive ecosystem of pre-built charts. However, Kustomize isn’t necessarily harder to use, it’s just built for a different purpose. Kustomize shines when making small, targeted modifications to existing resource definition files, making it ideal for managing configurations across multiple environments, rather than packaging whole applications like Helm.
+
+One of Helm’s [drawbacks](/blog/5-helm-shortcomings/) is that its dynamically populated templates can be hard to interpret unless rendered. In contrast, Kustomize manages and modifies plain YAML manifests, making it much easier to read and understand. Engineers are likely to find Kustomize configurations more approachable and intuitive compared to Helm charts when inspecting a repository.
+
+### Kustomize vs Helm: Packaging
+
+Helm is purpose-built for packaging and distributing Kubernetes applications, while Kustomize is more about applying modifications to existing YAML configurations. If you need full application lifecycle management, Helm might be better suited. For granular resource customization, Kustomize is the way to go.
+
+### Native kubectl integration
+
+Kustomize is natively integrated with [kubectl](/guides/kubectl/) from version 1.14 onwards, meaning you can use it directly without needing any additional tools. Helm, on the other hand, requires the Helm CLI.
+
+Install `kubectl` and run:
+```
+kubectl kustomize <kustomization_directory>
+```
+
+Helm on the other hand is custom built to be used in Kubernetes environmnets but it is not integrated into `kubectl`, nor is it considered “kubernetes-native“.
```

Comment:
Again, I'm not sure what kubernetes-native means here

Reply:
I removed the line